src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml

```diff
@@ -154,9 +154,10 @@
         <div class="govuk-!-margin-top-3">
 
             <div class="govuk-button-group">
+                @* This is a hidden button to force the default action to be save rather than submit*@
+                <button aria-hidden="true" class="govuk-visually-hidden" type="submit" name="Action" value="@ActionEnum.Save" tabindex="-1">Save</button>
```

Comment:
do you need to add  **data-prevent-double-click="true"** as well ?

Reply:
No this button isnt clickable...

Reply:
ok